Click into the grey part of the following cell, then run it by clicking "Cell >> Run" or the play button up above.

In [2]:
import sys; print 'python:\t\t\t', str(sys.version_info[0]) + '.' + str(sys.version_info[1])
print 'modules:'
import numpy;   print '\tnumpy:\t\t', numpy.__version__
import scipy;   print '\tscipy:\t\t', scipy.__version__
import pandas;  print '\tpandas:\t\t', pandas.__version__
import sklearn; print '\tsklearn:\t', sklearn.__version__

python:			2.7
modules:
	numpy:		1.11.1
	scipy:		0.18.1
	pandas:		0.18.1
	sklearn:	0.17.1


The output should be something like this:

`
python:2.7
modules:
    numpy:		1.11.1
    scipy:		0.18.1
    pandas:		0.18.1
    sklearn:	0.17.1
`

If it isn't, raise your hand.